test/integration/pairs_test.lua

```diff
@@ -58,20 +64,37 @@ g.before_each(function()
             end
         ]])
     end
-end)
+end
+
+g_memtx.before_all = function() before_all(g_memtx, 'memtx') end
+g_vinyl.before_all = function() before_all(g_vinyl, 'vinyl') end
+
+g_memtx.after_all = function() after_all(g_memtx) end
+g_vinyl.after_all = function() after_all(g_vinyl) end
+
+g_memtx.before_each(function() before_each(g_memtx) end)
+g_vinyl.before_each(function() before_each(g_vinyl) end)
+
+local function add(name, fn)
+    g_memtx[name] = fn
+    g_vinyl[name] = fn
+end
```

Comment:
Can we separate this code to some helper sometimes?

Reply:
Should be moved into separate patch

Reply:
Agree.